code/modules/awaymissions/gateway.dm

```diff
@@ -154,6 +154,9 @@ var/obj/machinery/gateway/centerstation/the_gateway = null
 		user << "\black The gate is already calibrated, there is no work for you to do here."
 		return
 
+/obj/machinery/gateway/centerstation/Destroy()

```

Comment:
You left the awful type pathing in. centerstation and centeraway have so much duped code. It's awful.